csharp/test/Drivers/Apache/Hive2/ClientTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+using Apache.Arrow.Adbc.Drivers.Apache.Hive2;
+using Xunit.Abstractions;
+
+namespace Apache.Arrow.Adbc.Tests.Drivers.Apache.Hive2
+{
+    public class ClientTests : Common.ClientTests<ApacheTestConfiguration, HiveServer2TestEnvironmentExtended>
```

Comment:
Please include a SampleBuilder, similar to https://github.com/apache/arrow-adbc/blob/1ce10b88ac33125dbd29e467208eadb721c105ef/csharp/test/Drivers/Apache/Impala/ImpalaTestEnvironment.cs#L139, to validate the types are returning as expected.